### ISSUE TITLE
After downgrading to 1.07 firmware, there is no sound from the right earphone

### ISSUE THREAD

**Issue by 1584598022** [kioipp/bose_earbuds_firmware#3]

**Comment by monkeycraps** [comment]
do u fix it ? i am using mac, which version of updater u used ? does it needed for edited updator from ced ?

**Comment by 1584598022** [comment]
> do u fix it ? i am using mac, which version of updater u used ? does it needed for edited updator from ced ?

No. Then I updated to 2.07, it can work rightly. You can use the latest updater from the website , then click https://www.bose.cn/ced/bose_music_china/lando/index.xml , you will see the official index in Charles ,use the edited index to replace it.

**Comment by monkeycraps** [comment]
i make it, at windows system.
when downgrade the firmware, did u map local the bin file @1584598022

**Comment by monkeycraps** [comment]
after downgrade, the ability of anti-voice is enhanced. and it is more comfortable.

**Comment by 1584598022** [comment]
> i make it, at windows system. when downgrade the firmware, did u map local the bin file @1584598022

I did map local the index not the bin file. I guess it's because of my factory firmware version is 1.29, so i can't rollback it to a lower one.

**Comment by congyoua** [comment]
same here downgraded to 1.0.7 and no sound from right earbud. then I updated back to 1.27, which is the factory default version, still no sound. Then I updated to 2.0.7 without any mapping just vanilla updater and everything works fine now. so I don't think it is a problem with the factory version.

**Comment by thinki** [comment]
I met the same issue as I downgrade to 1.0.7 with no sound on right earbuds. Also still no sound when I updated back to 1.2.7. And I can't upgrade to 2.0.7 on PC side with Bose Updater.
Fortunately I succeed to upgrade to 2.0.7 on Android App. And my earbuds factory firmware version is 1.2.10, so I agree with @1584598022 that rolling back to a lower version may cause problems. In addition, my earbuds is stone blue version, which is on sale at about Sep 2021, which may be too late version.
It's a pity that I can't experience the magic 1.0.7 version Active Noise Cancelling 😞  But really thanks @kioipp summary for this fantastic downgrade method. 😃

**Comment by Canawan** [comment]
> > do u fix it ? i am using mac, which version of updater u used ? does it needed for edited updator from ced ?
> 
> No. Then I updated to 2.07, it can work rightly. You can use the latest updater from the website , then click https://www.bose.cn/ced/bose_music_china/lando/index.xml , you will see the official index in Charles ,use the edited index to replace it.

Can you give more specific instructions on how to replace index in charles?  because I have never used this application before
